compose/app/build.gradle.kts

```diff
@@ -26,12 +26,19 @@ android {
 
     buildTypes {
         defaultConfig {
-            buildConfigField("String", "FW_CLIENT_ID", "\"f6d6ec1275217f178cdff91363825cb390e038c1168f64f6efa23cb95ec6b325\"")
+            buildConfigField(
+                "String",
+                "FW_CLIENT_ID",
+                "\"f6d6ec1275217f178cdff91363825cb390e038c1168f64f6efa23cb95ec6b325\""
```

Comment:
Is this a client id only for the demo app? I am concerned, but let's take it separate discussion.